contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -15,6 +15,10 @@ contract RaidenMicroTransferChannels {
     // Contract semantic version
     string public constant version = '1.0.0';
 
+    // We temporarily limit total token deposits in a channel to 100 tokens with 18 decimals.
+    // This is just for the bug bounty release.
```

Comment:
Please also mention here in a comment that this is just for RDN and with its current (as of 30/11/2017) price and should not be considered to be the same for other tokens.